### DISCUSSION TITLE
is there a version of the immich application for android 7?

### DISCUSSION THREAD

**Discussion by ipchesnokovaep** [immich-app/immich#8917] [Q&A] [answered]
is there a version of the immich application for android 7?
for older devices

**Comment by JW-CH** [comment, marked as answer]
The short answer is no, there is no 'official' version for android 7.

There are many helpful individuals on discord if you need help with the following:

Here is an answer from Discord from Alex (https://discord.com/channels/979116623879368755/1164485402489540608):
> Ah, recently Android advises us you update the api version for the build because of app store policy otherwise the app cannot be updated. You can clone the repository and build the apk for your phone, I think it will work OK

You could build the app yourself by modifying the `mobile/android/app/build.gradle`

```
compileSdkVersion 33     <--- don't change this
    ...
      defaultConfig {
          ...
          minSdkVersion 24          <--- decreased from v 26
          targetSdkVersion 33     <--- don't change this
          ...
      }
```

> For offline compilation, run flutter build apk from `mobile/` folder

**Reply by ipchesnokovaep** [reply]
Thank you
Perhaps someone has already compiled it? I'm not qualified to do this unfortunately.
Otherwise, my solution is to flash the smartphone with unofficial firmware with the desired version of Android

**Reply by JW-CH** [reply]
Just ask someone (preferably a team member) on the discord for an APK. But be cautious that APKs from an unofficial source could be modified.
